base/environments/jax_environment.py

```diff
@@ -284,17 +284,23 @@ def save_parameters(self):
         Also saves all parameters to the data file.
         """
         from jax.util.labrad import remove_labrad_units
-
-        params = {}
-        params_full = {}
         pb = self.cxn.parameter_bank
+        params_serialized, params_full_serialized = pb.get_frozen_parameters(self.scheduler.rid)
+        params = pickle.loads(params_serialized)
+        params_full = pickle.loads(params_full_serialized)
```

Comment:
change this name? It's unclear what params_full is (maybe cached_params or copied_params?)

Reply:
Also added docstrings to clarify.